tests/cstorvolume/provisioing/provisioning_utils.go

```diff
@@ -0,0 +1,208 @@
+package provisioning
+
+import (
+	"fmt"
+	"strconv"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	cstorapis "github.com/openebs/api/pkg/apis/cstor/v1"
+	"github.com/openebs/cstor-operators/tests/pkg/k8sclient"
+	"github.com/openebs/cstor-operators/tests/pkg/cspc/cspcspecbuilder"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	cstorCSIProvisionerName = "cstor.csi.openebs.io"
+)
+
+// ProvisionCSIVolume will provision Volume using CStor-CSI
+func ProvisionCSIVolume(pvcName, pvcNamespace, scName string) {
+	var (
+		// PVC will contains the PersistentVolumeClaim object created for test
+		pvc *corev1.PersistentVolumeClaim
+		// SC will contains the StorageClass object created for test
+		sc *storagev1.StorageClass
+	)
+
+	parameters := map[string]string{
+		"cas-type":         "cstor",
+		"cstorPoolCluster": cspc.Name,
+		"replicaCount":     strconv.Itoa(cstorsuite.ReplicaCount),
+	}
+	sc = createStorageClass(scName, parameters)
+
+	err := cstorsuite.client.CreateNamespace(pvcNamespace)
+	Expect(err).To(BeNil())
+
+	pvc = createPersistentVolumeClaim(pvcName, pvcNamespace, sc.Name)
+
+	err = cstorsuite.client.WaitForPersistentVolumeClaimPhase(
+		pvc.Name, pvc.Namespace, corev1.ClaimBound, k8sclient.Poll, k8sclient.ClaimBindingTimeout)
+	Expect(err).To(BeNil())
+
+	//TODO: Uncomment below code
+	// cvcSpecBuilder.SetCVCSpec(cvc)
```

Comment:
Does this need to be uncommented for this PR?

Reply:
Not required as of now

Reply:
It will more useful in scaleup and scaledown cases